test(engine): migrate remaining tests to new udf import paths

caidanw opened the pull request:
## Summary
- Update remaining python tests to use new engine udf import paths (replace legacy osprey_udf / osprey_executor / osprey_stdlib imports)
- No behavioral changes; import lines only

## Rationale
Align tests with refactored package structure under  and  to prevent import errors and keep consistency.

## Scope
Only touched test files' import statements (11 files). No production code changes.

## Validation
Pre-commit hooks (ruff, formatting) passed locally. Grep sweep confirms no lingering legacy import module paths in tests.

caidanw committed:
test(engine): migrate stdlib UDF tests to new import paths

caidanw committed:
test(executor): patch correct logger path in ExecutionContext tests

caidanw committed:
test(engine): update validator tests to new engine import paths

caidanw committed:
test(engine): migrate remaining tests to new udf import paths

BinaryFiddler commented on osprey_worker/src/osprey/engine/ast_validator/validators/tests/test_imports_must_not_have_cycles.py:
```diff
@@ -1,10 +1,10 @@
+from osprey.engine.stdlib.udfs.import_ import Import
+from osprey.engine.udf.registry import UDFRegistry
 
 from ....conftest import CheckFailureFunction, RunValidationFunction
```
maybe take this as an opportunity to move to absolute import paths?

caidanw replied:
done

caidanw committed:
refactor: relative test imports to be asbolute paths

caidanw committed:
fix: remove import on test with missing class

Need to figure out what do with this, leaving a comment so I can come
back to it
